t/api/05-machines.t

```diff
@@ -84,8 +84,9 @@ is_deeply(
 
 
 my $res = $t->post_ok('/api/v1/machines', form => {name => "testmachine"})->status_is(400);    # missing backend
+is($t->tx->res->json->{error}, 'wrong parameter: backend', 'miss backend');
 $res = $t->post_ok('/api/v1/machines', form => {backend => "kde/usb"})->status_is(400);        #missing name
-
+is($t->tx->res->json->{error}, 'wrong parameter: name', 'miss name');
```

Comment:
This would look nicer as `$t->json_is('/error', 'wrong parameter: name')`.

Reply:
Modified it.